### PR TITLE
C581-change loan due dates

### PR DESCRIPTION
**Pull request by tomleexx**
add new test scripts for C581

use this enviornment https://folio-snapshot-2-okapi.dev.folio.org to run scripts

testrail link
https://foliotest.testrail.io/index.php?/cases/view/581

Allure report
https://jenkins-aws.indexdata.com/job/folioRancher/job/folioTestingTools/job/runCypressTests/4470/allure/#

### CODE REVIEW

File: cypress/e2e/loans/loan-change-due-dates.cy.js [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+    }).then((instance) => {
+      cy.deleteItemViaApi(instance.items[0].id);
+      cy.deleteHoldingRecordViaApi(instance.holdings[0].id);
+      InventoryInstance.deleteInstanceViaApi(instance.id);
+    });
+    Users.deleteViaApi(checkOutUser.userId);
+  });
+  it(
+    'C581 Loan Details: Test change due date (TaaS)',
+    { tags: [TestTypes.smoke, DevTeams.volaris] },
```
Comment (nayimovag) [inline cypress/e2e/loans/loan-change-due-dates.cy.js:101]: wrong testType and devTeam

File: cypress.config.js [hunk 2]
```diff
@@ -91,6 +91,6 @@ module.exports = defineConfig({
 
       return config;
     },
-    baseUrl: 'https://folio-testing-cypress-diku.ci.folio.org',
+    baseUrl: 'https://folio-snapshot-2.dev.folio.org/',
```
Comment (nayimovag) [inline cypress.config.js:94]: please don't change cypress configurations

File: cypress/e2e/loans/loan-change-due-dates.cy.js [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+let servicePointId;
+
+describe('change loan due dates', () => {
+  before('create inventory instance', () => {
+    let source;
+
+    cy.createTempUser([
+      Permissions.loansAll.gui,
+      Permissions.checkoutAll.gui,
+      Permissions.requestsAll.gui,
```
Comment (nayimovag) [inline cypress/e2e/loans/loan-change-due-dates.cy.js:37]: wrong set of permissions

**Review by nayimovag — CHANGES_REQUESTED** [review]

**Review by manowarqqq — APPROVED** [review]

File: cypress/e2e/loans/loan-change-due-dates.cy.js (hunk 3 above)
Comment (tomleexx) [reply cypress/e2e/loans/loan-change-due-dates.cy.js]: remove request permission

**Comment by tomleexx** [comment]
Permissions: view loans, edit loans
User with at least one open loan for item with item status Checked out needed

according to preconditions,I just use loan.all permission and check out permission

**Review by ernestyedigaryan1 — APPROVED** [review]

**Review by ostapwd — APPROVED** [review]

**Review by ostapwd — APPROVED** [review]

**Pull request merged**